src/store/tests/AutomatedTester.java

```diff
@@ -9,7 +9,7 @@
 
 public class AutomatedTester {
 
-	private GroceryStore groceryStore;
+	private static GroceryStore groceryStore;
```

Comment:
What does changing it to static do?

Reply:
It was to be able to use it in main method, which is static. Please disregard that per my previous messages to you. It can be deleted eventually. Right now you can approve of it--it won't make any difference. I believe we could get rid of it in declarations altogether. We will not need (possibly) the groceryStore, just GroceryStore (as a class reference). It still remains to be seen, but right new it's OK.

Reply:
Never mind that. No big deal, per the previous comment.